data/templates/article-wrapper.mst

```diff
@@ -15,6 +15,7 @@
         <script type="text/javascript" src="resource:///com/endlessm/knowledge/data/templates/js/{{{.}}}"></script>
         {{/javascript-files}}
 
+        <script type="application/json" id="content-metadata">{{{content-metadata}}}</script>
```

Comment:
Actually you don't need the `JSON.parse` in this case - you are just generating code directly.